README.md

```diff
@@ -116,16 +116,16 @@ ArmCord is also available in [Pi-Apps](https://github.com/Botspot/pi-apps).
 - You are breaking [Discord ToS](https://discord.com/terms#software-in-discord%E2%80%99s-services) by using ArmCord, but no one has been banned from using it or any of the client mods included.
 
 ## Can I use this on anything other than ARM?
-- Yes! ArmCord should work normally under Windows, MacOS and Linux as long as it has Electron support.  
+- Yes! ArmCord should work normally under Windows, MacOS, and Linux as long as it has Electron support.  
 
-## How can I access settings?
-- Either right click on the tray icon and click `Open Settings` or open Discord settings and scroll the sidebar down until you see information about versions. Click on ArmCord version and the settings window will popout.
+## How can I access the settings?
+- Either right-click on the tray icon and click `Open Settings` or open Discord settings and scroll down the sidebar until you see information about versions. Click on the ArmCord version and the settings window will pop out.
 
 ## How does this work?   
-- We are using the official web app and wrapping it up in Electron. While you may think this is lame and done like thousands of times before, what makes us unique is that we actually strive for creating a customized experience. You can very easily load in themes and mods with no installers/injectors. You can even make the client have transparency effects and follow fluent design on Windows! At it's core, it's just a simple web wrapper, however we applied many patches to make this work well for you <3
+- We are using the official web app and wrapping it up in Electron. While you may think this is lame and done like thousands of times before, what makes us unique is that we actually strive for creating a customized experience. You can very easily load themes and mods with no installers/injectors. You can even make the client have transparency effects and follow the fluent design of Windows! At its core, it's just a simple web wrapper, however, we applied many patches to make this work well for you <3
```

Comment:
*it's

Reply:
If you're talking about "At its core" then it should be "its".

Chatgpt explanation (i am lazy):

"Its" is a possessive pronoun used to indicate that something belongs to or is associated with something else. It does not require an apostrophe, and the spelling remains the same regardless of whether it is used as a subject or an object in a sentence. For example: "The cat licked its paws" (possessive form of "it").

"It's" is a contraction of "it is" or "it has". It requires an apostrophe to indicate the omission of the letter "i" in "is" or "ha" in "has". For example: "It's raining outside" (contraction of "it is").